common/app/navigation/FooterLinks.scala

```diff
@@ -128,22 +128,22 @@ object FooterLinks {
   val usListThree = List(
     FooterLink("Advertise with us", "https://advertising.theguardian.com/us/advertising", "us : footer : advertise with us"),
     FooterLink("Guardian Labs", "/guardian-labs-us", "us : footer : guardian labs"),
-    FooterLink("Search jobs", "https://jobs.theguardian.com/jobs?INTCMP=NGW_FOOTER_US_GU_JOBS", "us : footer : jobs"),
+    FooterLink("Search jobs", "https://jobs.theguardian.com/?INTCMP=NGW_FOOTER_US_GU_JOBS", "us : footer : jobs"),
     FooterLink("Dating", "https://soulmates.theguardian.com/?INTCMP=soulmates_us_web_footer", "us : footer : soulmates"),
     discountCodes("us")
   )
 
   val auListThree = List(
     FooterLink("Guardian Labs", "/guardian-labs-australia", "au : footer : guardian labs"),
     FooterLink("Advertise with us", "https://advertising.theguardian.com/", "au : footer : advertise with us"),
-    FooterLink("Search UK jobs", "https://jobs.theguardian.com/jobs?INTCMP=NGW_FOOTER_AU_GU_JOBS", "au : footer : uk-jobs"),
+    FooterLink("Search UK jobs", "https://jobs.theguardian.com/?INTCMP=NGW_FOOTER_AU_GU_JOBS", "au : footer : uk-jobs"),
```

Comment:
It links fine but I don't know if there's SEO implications of leaving the `/` so might want to remove it so the links are `https://jobs.theguardian.com?INTCMP=NGW_FOOTER_AU_GU_JOBS`

Reply:
👍